deep_daze/deep_daze.py

```diff
@@ -408,13 +512,13 @@ def generate_gif(self):
             if file_name.startswith(self.textpath) and file_name != f'{self.textpath}.jpg':
                 images.append(imread(os.path.join('./', file_name)))
 
-        mimsave(f'{self.textpath}.gif', images)
-        print(f'Generated image generation animation at ./{self.textpath}.gif')
+        mimsave(f'{self.textpath}.mp4', images)
+        print(f'Generated image generation animation at ./{self.textpath}.mp4')
```

Comment:
Perhaps this mp4 generating code should be based on a conditional with a new param like `save_video` rather than replacing the `save_gif` functionality.

I prefer mp4s, but someone might want to generate a gif for whatever reason

Reply:
Fair enough, I can add a separate `save_video` flag